### PR TITLE
Add team radar for Shared Services

### PR DESCRIPTION
**Pull request by benjschneider**

### CODE REVIEW

**Comment by maxdanilov** [comment]
@benjschneider you would need at least 1 review (preferably from someone in your team) to merge this.

**Comment by benjschneider** [comment]
> @benjschneider you would need at least 1 review (preferably from someone in your team) to merge this.

On it!

File: shared.csv [hunk 1]
```diff
@@ -1 +1,24 @@
...
+React,Languages & Frameworks,Trial,TRUE,
+Redux,Languages & Frameworks,Trial,TRUE,
+Gradle,Tools,Trial,TRUE,
+Nimbus,Languages & Frameworks,Trial,TRUE,
+Postgresql,Infrastructure Components,Trial,TRUE,
+GraphQL,Languages & Frameworks,Assess,TRUE,At the moment there is no use case for our application but we'd like to use it at some point
+React i18next,Languages & Frameworks,Trial,TRUE,
+kustomize,Tools,Trial,TRUE,
+Kubernetes,Infrastructure Components,Trial,TRUE,
+Google CloudBuild,Services,Trial,TRUE,
```
Comment (dlavrenuek) [inline shared.csv]: According to official page the naming is "Google Cloud Build" https://cloud.google.com/cloud-build

**Review by dlavrenuek — APPROVED** [review]

File: shared.csv (hunk 1 above)
Comment (benjschneider) [reply shared.csv]: Good catch, changed it!

**Pull request merged**